Cargo.toml

```diff
@@ -222,6 +239,7 @@ sp-genesis-builder                         = { git = "https://github.com/parityt
 sp-inherents                               = { git = "https://github.com/paritytech/polkadot-sdk", branch = "stable2407", default-features = false }
 sp-io                                      = { git = "https://github.com/paritytech/polkadot-sdk", branch = "stable2407", default-features = false }
 sp-keystore                                = { git = "https://github.com/paritytech/polkadot-sdk", branch = "stable2407", default-features = false }
+sp-mmr-primitives                          = { git = "https://github.com/paritytech/polkadot-sdk", branch = "stable2407", default-features = false }
```

Comment:
Not needed

Reply:
ImpRuntimeApi uses this dependency.

Reply:
I see, we'll remove it in future versions